### PR TITLE
Version 1.0.18+1

### PR DESCRIPTION
**Pull request by naveen-egov**

### CODE REVIEW

File: apps/health_campaign_field_worker_app/lib/widgets/progress_bar/beneficiary_progress.dart [hunk 1]
```diff
@@ -59,7 +60,8 @@ class _BeneficiaryProgressBarState extends State<BeneficiaryProgressBar> {
             .where((element) =>
                 element.dateOfRegistrationTime.isAfter(gte) &&
                 (element.isDeleted == false || element.isDeleted == null) &&
-                element.dateOfRegistrationTime.isBefore(lte))
+                element.dateOfRegistrationTime.isBefore(lte) &&
+                element.auditDetails!.createdBy == context.loggedInUserUuid)
```
Comment (vishal-egov) [inline apps/health_campaign_field_worker_app/lib/widgets/progress_bar/beneficiary_progress.dart:64]: why do we need to filter it multiple times, it is already fetching based on user id during search

File: apps/health_campaign_field_worker_app/lib/pages/home.dart [hunk 2]
```diff
@@ -535,23 +535,23 @@ class _HomePageState extends LocalizedState<HomePage> {
...
-    //             builder: (context) => DriftDbViewer(
-    //               context.read<LocalSqlDataStore>(),
-    //             ),
-    //           ),
-    //         );
-    //       },
-    //     ),
-    //   ],
-    // );
+    homeItems.addAll(
```
Comment (vishal-egov) [inline apps/health_campaign_field_worker_app/lib/pages/home.dart:538]: do not push this keep it in localitself

File: apps/health_campaign_field_worker_app/lib/blocs/search_households/search_households.dart [hunk 3]
```diff
@@ -92,9 +92,7 @@ class SearchHouseholdsBloc
     final interventionDelivered = tasks
         .map(
           (task) {
-            return task.resources?.where((element) {
-              return element.auditDetails?.createdBy == userUid;
-            }).map(
+            return task.resources?.map(
```
Comment (vishal-egov) [inline apps/health_campaign_field_worker_app/lib/blocs/search_households/search_households.dart:95]: So we are showing count irrespective of user in search screen right?

**Review by vishal-egov — CHANGES_REQUESTED** [review]

**Comment by naveen-egov** [comment]
1.  Progress bar in home screen is user based 
2. Matrix in search beneficiary screen  is for device based

**Review by vishal-egov — APPROVED** [review]

**Review by kavi-egov — APPROVED** [review]

**Pull request merged**